### DISCUSSION TITLE
Multi-ragapps Deployment not working

### DISCUSSION THREAD

**Discussion by Rauljauregi** [ragapp/ragapp#229] [Q&A] [answered]
I have deployed an instance with multi-ragapps into production on a server. When I add a new app, there are problems, and it doesn't allow me to have two running on the same time. Finally I had to delete the one so one of them to work.

I really like the concept of Multi-Ragapp, congratulations, but I can't get it to work.

Solved: I had an issue in my end, the Multi-ragapps are working perfectly even in a server.

**Comment by benjaminbascary** [comment]
Can I get details on which cloud and which services are you using to deploy? I have been struggling a couple of days before being able to leave it running properly

**Comment by Rauljauregi** [comment]
I am using a VM instance on Google Cloud to deploy. I suspect it might be related to a firewall rule—maybe HTTP access needs to be added. It can be also the issue related to Docker instalation. If i can help let me know please

**Comment by benjaminbascary** [comment]
How much memory and storage have you assigned to the VM?

**Comment by Rauljauregi** [comment]
It is a 10Gb e2-medium instance. I have to clean temporal files, and docker cache and prune image and containers, in case I have to create images and containers again. to have enough memory.

It was not easy at first but now cleaning everything before, it works perfectly.

**Comment by benjaminbascary** [comment, marked as answer]
Your memory is too low for Ragapp.. you should go for 16 gb minimum because a couple of ragapp instances can deplete your memory.

Be sure to check the storage space in the mounted disk, otherwise it is going to fail when creating new containers

**Comment by Rauljauregi** [comment]
Thank you, yes, but for the moment I don´t have Ragapp in production. I am just checking and testing the application (I like it so much for the moment). You are right I may need to rise the amount of memory in case I go to production.

And yes the memory was the issue I have when trying to have more thn one Ragapp running.

Now I need the pdf files that are linked as sources to be opened. Which in my case they are not opening. Do you achieved to open those files?
